functional_tests.go

```diff
@@ -2715,6 +2715,385 @@ func testCopyObject() {
 	successLogger(testName, function, args, startTime).Info()
 }
 
+// Tests SSE-C get object ReaderSeeker interface methods.
+func testEncryptedGetObjectReadSeekFunctional() {
+	// initialize logging params
+	startTime := time.Now()
+	testName := getFuncName()
+	function := "GetObject(bucketName, objectName)"
+	args := map[string]interface{}{}
+
+	// Seed random based on current time.
+	rand.Seed(time.Now().Unix())
+
+	// Instantiate new minio client object.
+	c, err := minio.New(
+		os.Getenv(serverEndpoint),
+		os.Getenv(accessKey),
+		os.Getenv(secretKey),
+		mustParseBool(os.Getenv(enableHTTPS)),
+	)
+	if err != nil {
+		logError(testName, function, args, startTime, "", "Minio client object creation failed", err)
+		return
+	}
+
+	// Enable tracing, write to stderr.
+	// c.TraceOn(os.Stderr)
```

Comment:
Maybe we can remove disabled code?

Reply:
No, it is kept on purpose for debugging.

Reply:
Yes, but if someone wants to debug he/she can simple add. However very minior - if you think we should keep, than we can...

Reply:
We have kept it in all SDKs

Reply:
Okay